gwpy/utils/lal.py

```diff
@@ -70,6 +71,9 @@
 
 LAL_TYPE_STR_FROM_NUMPY = {k: LAL_TYPE_STR[v] for
                            (k, v) in LAL_TYPE_FROM_NUMPY.items()}
+LAL_NUMPY_FROM_TYPE_STR = {v: k for k, v in LAL_TYPE_STR_FROM_NUMPY.items()}
+
+REGEX_LAL_DTYPE = re.compile(r'(U?INT|REAL|COMPLEX)\d+')
```

Comment:
Can you please rename this to appease my pedantry?

```suggestion
LAL_TYPE_REGEX = re.compile(r'(U?INT|REAL|COMPLEX)\d+')
```

Reply:
Done.